app/src/app/api/v0/inventory/[inventory]/download/route.ts

```diff
@@ -51,12 +51,22 @@ export const GET = apiHandler(async (req, { params, session }) => {
         as: "inventoryValues",
         include: [
           {
-            model: db.models.GasValue,
-            as: "gasValues",
+            model: db.models.ActivityValue,
+            as: "activityValues",
             include: [
-              { model: db.models.EmissionsFactor, as: "emissionsFactor" },
+              {
+                model: db.models.GasValue,
+                as: "gasValues",
+                include: [
+                  { model: db.models.EmissionsFactor, as: "emissionsFactor" },
+                ],
+              },
             ],
           },
+          {
+            model: db.models.GasValue,
+            as: "gasValues",
```

Comment:
These will not contain any data with our current way to save the manual input data. So we can either fix that later or remove/ comment out this block for now